### PR TITLE
chore(roadmap): update roadmap for 2.x

### PR DESCRIPTION
**Pull request by slack**
Closes https://github.com/deis/workflow/issues/309

Linked to related issues from deis/deis for roadmap items where it makes sense. Many of these roadmap items still need Workflow/Kubernetes-focused proposals.

### CODE REVIEW

File: src/roadmap/roadmap.md [hunk 1]
```diff
@@ -1,24 +1,110 @@
...
+## Private Registry and Native IaaS Registry Support
 
-* [Alpha Issues](https://github.com/deis/deis/issues/4743)
-* [Beta Issues](https://github.com/deis/deis/issues/4809)
-* [Release Candidate Issues]()
+Many users of Deis v1 requested the ability to push and pull application
+artifacts from private Docker registries. While v1 supported a
+[workaround](https://github.com/deis/deis/issues/2232) or two, operators
+usually had to intervene or build custom tooling. Deis Workflow aims to make it
+simple to configure your Worfklow install to pull and push images to private
```
Comment (mboersma) [inline src/roadmap/roadmap.md]: typo: "Workflow"

File: src/roadmap/roadmap.md [hunk 2]
```diff
@@ -1,24 +1,110 @@
-# Deis Roadmap
+# Deis Workflow Roadmap
 
-The Deis Roadmap is a community document created as part of the open [Planning Process][].
-Each roadmap item describes a high-level capability or grouping of features
-that are deemed important to the future of Deis.
+The Deis Workflow Roadmap is a community document created as part of the open
+[Planning Process][].  Each roadmap item describes a high-level capability or
```
Comment (sgoings) [inline src/roadmap/roadmap.md]: There's no link for this...

File: src/roadmap/roadmap.md [hunk 3]
```diff
@@ -1,24 +1,110 @@
...
+
+* [ ] deis/controller emitting metrics
+* [ ] Define and deliver alerts with Kapacitor: <https://github.com/deis/monitor/issues/44>
+
+## Workflow Addons/Services
+
+Developers should be able to quickly and easily provision application
+dependencies using a services or addon abstraction.
+
+* [ ] <https://github.com/deis/deis/issues/231>
```
Comment (sgoings) [inline src/roadmap/roadmap.md]: Could you just roll this checkbox item into the sentence above?

File: src/roadmap/roadmap.md [hunk 4]
```diff
@@ -1,24 +1,110 @@
...
+* [ ] Application Autoscaling <https://github.com/deis/workflow/issues/403>
+
+## Interactive `deis run /bin/bash`
+
+Provide the ability for developers to launch an interactive terminal session in
+their application environment.
+
+Related issues:
+
+* [ ] <https://github.com/deis/workflow-cli/issues/98>
```
Comment (sgoings) [inline src/roadmap/roadmap.md]: In some places "Related issues" are bullets... and in other places they're check boxes. I'd like to see a checkbox for the feature, and related issues be bullets... we might have lots of GH issues/PRs that are assembled for things like `deis run`

File: src/roadmap/roadmap.md [hunk 5]
```diff
@@ -1,24 +1,110 @@
...
+* [ ] <https://github.com/deis/deis/issues/117>
+
+## Log Streaming
+
+Stream application logs via `deis logs -f`:
+
+* [ ] <https://github.com/deis/deis/issues/465>
```
Comment (sgoings) [inline src/roadmap/roadmap.md]: Any way this could be merged with the above sentence as well?

**Comment by sgoings** [comment]
Thanks for making those fixes @slack!

**Comment by slack** [comment]
No problemo!

**Pull request merged**